x-eithert/src/X/Control/Monad/Trans/Either.hs

```diff
@@ -204,3 +211,28 @@ joinErrorsEither =
 tryEitherT :: (Functor m, MonadCatch m, Exception e) => (e -> x) -> m a -> EitherT x m a
 tryEitherT handler = firstEitherT handler . newEitherT . try
 {-# INLINE tryEitherT #-}
+
+-- | Lift an 'Either' into 'Errors'.
+eitherErrors :: Either e a -> Errors e a
+eitherErrors e =
```

Comment:
Export y/n?

Honestly the `Errors` type is only mildly useful because it's limited to applicative. It's also a type synonym, so I prefer not to use it directly.

If you happen to want to use it, you'd probably want this function, else you have to learn how Lift works.